src/style.css

```
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: normal;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 10px;
+
+}
```

```suggestion
.form {
  margin-top: 20px;
}
```
This horisontal margin cause your tests to fail, i suppose